connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/LoggingResource.java

```diff
@@ -60,62 +60,43 @@
     @GET
     @Path("/")
     public Response listLoggers() {
-        Map<String, Map<String, String>> loggers = new TreeMap<>();
-        Enumeration<Logger> enumeration = currentLoggers();
-        Collections.list(enumeration)
-                .stream()
-                .filter(logger -> logger.getLevel() != null)
-                .forEach(logger -> loggers.put(logger.getName(), levelToMap(logger)));
-
-        Logger root = rootLogger();
-        if (root.getLevel() != null) {
-            loggers.put(ROOT_LOGGER_NAME, levelToMap(root));
-        }
+        Map<String, Map<String, String>> loggers = currentLoggers()
+            .stream()
+            .map(logger -> new AbstractMap.SimpleEntry<>(logger.getName(), configLevelToMap(logger)))
+            .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
```

Comment:
Is the `.map()` needed?

```suggestion
            .collect(Collectors.toMap(logger -> logger.getName(), logger -> configLevelToMap(logger));
```

Reply:
Also, `Collectors.toCollection(Treeset::new)` might be useful

Reply:
Great. I will apply it. :)